deploy/nginx/nginx.conf

```diff
@@ -52,7 +52,65 @@ http {
+        rewrite ^/scheduler/kubernetes/dynamic-provisioning.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/dynamic-provisioning permanent;
+        rewrite ^/scheduler/kubernetes/elasticstack-kibana-k8s.html /portworx-install-with-kubernetes/application-install-with-kubernetes/elastic-search-and-kibana permanent;
+        rewrite ^/scheduler/kubernetes/encrypted-pvc-awskms.html /portworx-install-with-kubernetes/storage-operations/create-snapshots permanent;
+        rewrite ^/scheduler/kubernetes/encrypted-pvc-ibm-kp.html /portworx-install-with-kubernetes/cloud/ibm permanent;
+        rewrite ^/scheduler/kubernetes/encrypted-pvc-vault.html /key-management/vault permanent;
+        rewrite ^/scheduler/kubernetes/encrypted-volumes.html /portworx-install-with-kubernetes/storage-operations/create-pvcs/create-encrypted-pvcs/ permanent;
+        rewrite ^/scheduler/kubernetes/etcd-certs-using-secrets.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/etcd-certs-using-secrets permanent;
+        rewrite ^/scheduler/kubernetes/failover.html /portworx-install-with-kubernetes/application-install-with-kubernetes/test-failover permanent;
+        rewrite ^/scheduler/kubernetes/going-production-with-k8s.html /portworx-install-with-kubernetess/operate-and-maintain-on-kubernetes/running-in-production permanent;
+        rewrite ^/scheduler/kubernetes/index.html /portworx-install-with-kubernetes permanent;
+        rewrite ^/scheduler/kubernetes/install-cfcr-etcd-release.html /portworx-install-with-kubernetes permanent;
+        rewrite ^/scheduler/kubernetes/install-legacy.html /portworx-install-with-kubernetes permanent;
+        rewrite ^/scheduler/kubernetes/install-pks-vsphere-local.html /portworx-install-with-kubernetes/on-premise/install-pks permanent;
+        rewrite ^/scheduler/kubernetes/install-pks-vsphere-shared.html /portworx-install-with-kubernetes/cloud/install-pks permanent;
+        rewrite ^/scheduler/kubernetes/install-pks-vsphere.html /portworx-install-with-kubernetes/on-premise/install-pks permanent;
+        rewrite ^/scheduler/kubernetes/install-pks.html /portworx-install-with-kubernetes/on-premise/install-pks permanent;
+        rewrite ^/scheduler/kubernetes/install.html /portworx-install-with-kubernetes permanent;
+        rewrite ^/scheduler/kubernetes/k8s-node-decommission.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/decommission-a-node permanent;
+        rewrite ^/scheduler/kubernetes/k8s-node-maintenance.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/enter-maintenance-mode permanent;
+        rewrite ^/scheduler/kubernetes/k8s-node-rejoin.html /portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/decommission-a-node permanent;
+        rewrite ^/scheduler/kubernetes/k8s-px-app-samples.html /portworx-install-with-kubernetes permanent;
```

Comment:
/scheduler/kubernetes/k8s-px-app-samples.html to /portworx-install-with-kubernetes/application-install-with-kubernetes/